### ISSUE TITLE
Improve color contrast of NED/SIMBAD/etc info in Aladin window in black hole mode

### ISSUE THREAD

**Issue by ale-munozarancibia** [alercebroker/ztf_explorer#313]
Although the column names appear in black, the info in white characters is hard to read in black hole mode (in "normal" mode works fine)

Dark:
![Screenshot from 2021-03-22 15-34-42](https://user-images.githubusercontent.com/42121847/112040841-713f7600-8b24-11eb-9716-25ef4aa6eeac.png)
Non-dark:
![Screenshot from 2021-03-22 15-35-09](https://user-images.githubusercontent.com/42121847/112040862-77355700-8b24-11eb-9195-3b726069de82.png)
